test/compilable/issue17167.sh

```diff
@@ -18,6 +17,6 @@ echo 'void main() {}' > "${src}"
 # Only compile, not link, since optlink can't handle long file names
 $DMD -m"${MODEL}" "${DFLAGS}" -c -of"${bin}" "${src}" || exit 1
 
-rm -rf "${dir:?}"/"$TEST_NAME"
+rm -rf "${RESULTS_OUTPUT}"
 
 echo Success
```

Comment:
This is superfluous too, but already removed in https://github.com/dlang/dmd/pull/8025